templates/web/fixmystreet.com/index.html

```diff
@@ -0,0 +1,20 @@
+[% map_js = PROCESS 'front/javascript.html' %]
```

Comment:
Ah I see, it still could be shown on other pages (e.g. faq, privacy, reports, contact). Hmm, perhaps an IF statement in footer-marketing then, rather than empty; I still would not like to override index.html only to remove pagefooter.